tests/system/source/main.d

```diff
@@ -97,8 +87,8 @@ private void checkBlockHeight (ulong height)
     // TODO: This is a hack because of issue #312
     // https://github.com/bpfkorea/agora/issues/312
     API[NodeCnt] clients;
-    foreach (idx, const ref addr; Addrs)
-        clients[idx] = new RestInterfaceClient!API(addr.withSchema());
+    foreach (idx, const ref addr; addresses)
```

Comment:
You have to pass it as argument to this function

Reply:
I thought so, too. That'd be fine.